content/en/docs/TTL-CleanUp/label_format.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Label Format
+description: Supported format for the label.
+weight: 30
+---
+
+The ttl controller is purposefully engineered to exclusively manage resources equipped with a specific label: **cleanup.kyverno.io/ttl**. This label serves to establish a time-to-live duration for a resource, effectively functioning as a countdown timer from the resource's creation time. The value assigned to this label is represented as a string and can be expressed in minutes, hours, or days. For instance, examples of valid labels include: `cleanup.kyverno.io/ttl: 30m` and `cleanup.kyverno.io/ttl: 30d`. Notably, this label system also accommodates the provision of an absolute date and time for when the cleanup process should occur. These absolute timestamps adhere to the ISO 8601 standards, as demonstrated by labels like `cleanup.kyverno.io/ttl: 2022-08-04T00:30:00Z` and `cleanup.kyverno.io/ttl: 2022-09-30`.
```

Comment:
@VedRatan, @eddycharly: this cannot be a valid value for a label: `2022-08-04T00:30:00`. How is this provided as the value of the `cleanup.kyverno.io/ttl` label?

Reply:
Yes, we need to remove the semicolon, sorry for that pushing the fix right now.